package.json

```diff
@@ -14,7 +14,8 @@
     "start:prod": "neutrino build && node .",
     "start:debugger": "neutrino build && node --inspect .",
     "precommit": "lint-staged",
-    "heroku-postbuild": "npm run build"
+    "heroku-postbuild": "npm run build",
+    "fetchNimbledroidData": "neutrino build && node build/nimbledroid.js"
```

Comment:
I don't think the `neutrino build` should be here. The build will be run as part of the Heroku slug compile (see Heroku docs on lifecycle) and the slug is then used for any future commands (whether that be `heroku run`, the scheduler, a web dyno or a worker dyno).